.travis.yml

```diff
@@ -8,15 +8,36 @@ sudo: false
 addons:
   apt:
     packages:
+    - bc
+    - doxygen
+    - expect
+    - gettext
+    - libncurses5-dev
     - tree
-    - fish
+
+env:
+  global:
+    - PATH="$HOME/fish/bin:$PATH"
+    - OMF_REPO_URI="https://github.com/$TRAVIS_REPO_SLUG"
+    - OMF_REPO_BRANCH="$TRAVIS_BRANCH"
+
+before_install:
+  - tests/install_fish.sh
```

Comment:
So we were run oh the tests in an old version of fish until now? 

Tools does sound better